common/unsafe/src/main/java/org/apache/spark/unsafe/bitset/BitSetMethods.java

```diff
@@ -45,6 +46,14 @@ public static void set(Object baseObject, long baseOffset, int index) {
     Platform.putLong(baseObject, wordOffset, word | mask);
   }
 
+  public static void setBlock(MemoryBlock base, long baseOffset, int index) {
```

Comment:
It is better to add doc for it. It is not clear to know what purpose of this method.

Reply:
Let me drop this since it is not used for now. It is provided for future use.